projects/storefrontlib/src/cms-components/checkout/components/shipping-address/shipping-address.component.spec.ts

```diff
@@ -197,11 +198,19 @@ describe('ShippingAddressComponent', () => {
         address = data;
       })
       .unsubscribe();
+
     expect(address).toBe(mockAddresses);
+
     component.cards$
       .subscribe(cards => {
         expect(cards.length).toEqual(2);
-        expect(cards[1].card.header).toBe('addressCard.selected');
+      })
+      .unsubscribe();
+
+    component.selectedAddress$
+      .pipe(takeWhile(selectedAddress => selectedAddress !== null))
+      .subscribe(selectedAddress => {
+        expect(selectedAddress).toEqual(mockAddress2);
```

Comment:
unfortunately, "expects" in subscribe are not called in synchronous tests.

Reply:
Tests have been fixed